explainer.md

```diff
@@ -607,14 +607,24 @@ enum XREye {
   "right"
 };
 
+[SecureContext, Exposed=Window,
+ Constructor(optional DOMPointInit translation, optional DOMPointInit orientation)]
+interface XRRigidTransform {
+  readonly attribute DOMPointReadOnly translation;
+  readonly attribute DOMPointReadOnly orientation;
```

Comment:
For these two attributes, should we pick a consistent naming pair?  To me, the two consistent pairs would be the following:
* `position`/`orientation`
* `translation`/`rotation`

The former implies a location within a base space, while the latter implies an explicit mutation of an entity's original transform.  Of these, the former makes a lot more sense to me as used in `...Pose` types like `XRViewerPose` and `XRInputPose`, as well as general world geometry types like `XRRay` - the developer will likely think of an entity's pose as a position and orientation within the general coordinate space in which they operate.

The key place that naming _could_ feel awkward is `XRSpace.getTransformTo`.  However, I've found that method name to often be confusing anyway - I, at least, need to do the mental exercise each time to remember which space should be the "from" space and which space should be the "to" space.  If we rename that method to `XRSpace.locateWithin(baseSpace)` or such, it could both make which space is which more clear and align even better with general `position`/`orientation` naming on `XRRigidTransform`.

Reply:
Thanks for the detailed explanation. I don't think I've ever given it much thought, but when laid out this way I do agree with your interpretation of the terms. 'position/orientation' feel like appropriate names, so I'll make that change.

As for `XRSpace.getTransformTo()` we already know we need to refactor that so that it takes the frame into account somehow, so it shouldn't be a big deal to tweak the naming to be more fitting.